docs/general/profiles.md

```diff
@@ -0,0 +1,110 @@
+
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## Frame types
+
+**Description:** Describes the origin of a single frame in a Profile.
+
+<!-- semconv profile.frame(full) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`profile.frame.type`](/docs/attributes-registry/profile.md) | string | Describes the interpreter or compiler of a single frame. | `python` | `Opt-In` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`profile.frame.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `dotnet` | [.Net](https://wikipedia.org/wiki/.NET) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `jvm` | [JVM](https://wikipedia.org/wiki/Java_virtual_machine) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `kernel` | [Kernel](https://wikipedia.org/wiki/Kernel_(operating_system)) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `native` | [C](https://wikipedia.org/wiki/C_(programming_language)), [C++](https://wikipedia.org/wiki/C%2B%2B), [Go](https://wikipedia.org/wiki/Go_(programming_language)), [Rust](https://wikipedia.org/wiki/Rust_(programming_language)) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `perl` | [Perl](https://wikipedia.org/wiki/Perl) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `php` | [PHP](https://wikipedia.org/wiki/PHP) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `python` | [Python](https://wikipedia.org/wiki/Python_(programming_language)) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `ruby` | [Ruby](https://wikipedia.org/wiki/Ruby_(programming_language)) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `v8` | [V8](https://wikipedia.org/wiki/V8_(JavaScript_engine)) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## Reserved attributes
+
+**Description:** For cross tooling compatibility the following attributes SHOULD be avoided.
+
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `pprof::*` | string | Special indicator for [pprof](https://github.com/google/pprof). | `pprof::base` |
```

Comment:
semconv allows [a-z0-9_\.] chars https://github.com/open-telemetry/semantic-conventions/issues/1124, we also have no tooling support to reserve the namespace at the moment.

But also, who/how/when will use these attributes? what would they be stamped on?

Reply:
created https://github.com/open-telemetry/semantic-conventions/issues/1197 to track reserving namespace

Reply:
Thanks for pointing out the allowed array of characters`[a-z0-9_.]` in Attributes. I wasn't aware of that, when writing this. Is there another way to express colons?

Compatibility between OTel profiling and pprof (tool + protocol) is important to the OTel profiling SIG. pprof is a well established tool and protocol in the industry. And there are special features, like `pprof::*` that pprof (tool) interprets in a special way. To avoid conflicts and guarantee compatibility, `pprof::*` should be avoided.

I'm aware that this is a special case and to my knowledge, semantic convention does not cover such special cases at the moment. This is also the reason, this section is written manually here and not automatically generated like other attributes.

Reply:
This compatability section around google/pprof got removed.